ansible/TESTCA.md

```diff
@@ -101,23 +115,36 @@ Both services use nginx as a proxy to redirect the requests to listened ports:
 
 ---------------------------------------------
 
-## 6. Configuring the central server to use the test-CA
+## 7. About the ACME services
+
+ACME services are provided by acme2certifier which is a python app served on `port 8887` over nginx and uwsgi. 
+It uses `extensions_auth` and `extensions_sign` blocks in `CA.cnf` file when creating authentication and signing certificates respectively.  
+There are two methods to let the ACME server know which type of certificate to return:
+- profile id in `HTTP_USER_AGENT` header. To use this, put `eab_profiling: False` in `acme_srv.cfg` and use keyid_1 in security server's `acme.yml`. In addition, auth and sign profile id-s need to set in Central Server for the `Test CA`.
```

Comment:
What are the values that need to be set for the profile id-s on the Central Server?